Add a series of short blog posts (#311)
voxpupuli/voxpupuli.github.io: _posts/2023-06-09-module-deprecation.md

---
layout: post
title: Vox Pupuli Changelog - Deprecating old modules

> date: 2023-07-02

Reviewer: this date doesn't match the filename

Author: fixed

github_username: bastelfreak
twitter_username: bastelsblog
---

Hi!

> We had some discussions on IRC and on the mailinglist. We will archive the

Reviewer: ```suggestion
As a result of a set of discussions on IRC and on the mailinglist. We will archive the
```

> following modules:

Reviewer: ```suggestion
following modules due to lack of community engagement:
```

---
layout: post
title: Vox Pupuli Changelog - Deprecating old modules
date: 2023-07-02
github_username: bastelfreak
twitter_username: bastelsblog
---

Hi!

This is our new blog series, where we inform you about changes within Vox Pupuli itself or our codebase!

We had some discussions on IRC and on the mailinglist. We will archive the
following modules:

* https://github.com/voxpupuli/puppet-system
* https://github.com/voxpupuli/puppet-windows_autoupdate
* https://github.com/voxpupuli/puppet-windows_eventlog
* https://github.com/voxpupuli/puppet-windows_power
* https://github.com/voxpupuli/puppet-windows_env
* https://github.com/voxpupuli/puppet-sslcertificate
* https://github.com/voxpupuli/puppet-msoffice

The modules will stay available on Github and on the Forge. They will be marked
as deprecated on the forge. We can unarchive the repos any time in case someone

> steps up and wants to keep maintainging them.

Reviewer 2: ```suggestion
steps up and wants to keep maintaining them.
```
